composer.json

```diff
@@ -32,24 +32,23 @@
     },
     "require": {
         "php": "~8.1.0 || ~8.2.0 || ~8.3.0 || ~8.4.0",
+        "laminas/laminas-eventmanager": "^3.14",
+        "laminas/laminas-modulemanager": "^2.17",
         "laminas/laminas-serializer": "^3.0",
-        "laminas/laminas-servicemanager": "^4.0",
+        "laminas/laminas-servicemanager": "^4.4.0",
```

Comment:
Making Laminas Service Manager a hard dep is the right thing to do - I hate soft deps, but, Hydrator ships a standalone container specifically to avoid installation of servicemanager, so I'm not sure this is ideal. WDYT @froschdesign?